From 9ce665e76a78d2995496253cae30750bb00a0557 Mon Sep 17 00:00:00 2001
From: Yanqing Jing <michael.origin.j@gmail.com>
Date: Mon, 9 Sep 2024 20:18:40 +0800
Subject: [PATCH] update keyboard hwdb to add acer Predator PHN16-72

(cherry picked from commit d8b4be38ddc5e7cb1dc5e6da9b6e247d6a3d9ffd)

Resolves: RHEL-49810
---
 hwdb.d/60-keyboard.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index a72f619e7c..3bf4860e1d 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -231,6 +231,10 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnPredatorPHN16-71:*
  KEYBOARD_KEY_f5=prog1                                  # "predator sense" button
  KEYBOARD_KEY_66=micmute                                # Microphone mute button
 
+# Predator PHN16-72
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnPredatorPHN16-72:*
+ KEYBOARD_KEY_66=micmute                                # Microphone mute button
+
 # Nitro AN515-58
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnNitro*AN*515-58:pvr*
  KEYBOARD_KEY_8a=f20                                    # Microphone mute button
